### ISSUE TITLE
Support for 0.8.x?

### ISSUE THREAD

**Issue by marcstraube** [gsimon2/playlist-drag-and-drop#1]
Hey! I really like this module, as its making the placement of ambient sounds very easy!
Will you update it for support of 0.8.x?

Dragging from the sidebar works as intended, but dropping on the canvas results in an error:

```
playlist-drag-and-drop.js:49 playlist-drag-and-drop | Failed to get sound object from soundId: HhaAWvrXacruo2Im. Error: TypeError: Cannot read property 'path' of undefined
playlist-drag-and-drop.js:26 Could not make dataSoundId HhaAWvrXacruo2Im to a file path
```

**Comment by gsimon2** [comment]
Yea the update to `0.8.x` changed some data structures and I just got around to updating my copy Foundry yesterday. I will debug whats going on here when I get the chance and keep you updated. Thanks for posting!

**Comment by marcstraube** [comment]
Thanks for your fast answer and your work!

**Comment by gsimon2** [comment]
Just released version `1.1.0` which should be compatible with Foundry `0.8.x` (tested on `0.8.6`). Please re-open this issue if you still experience any problems. Thanks for posting!

**Comment by marcstraube** [comment]
I've already tested as soon as I received the notification mail. Works like a charm! Thanks for the fast fix!
